transcom/mymove
# B-19535-INT PrimeEstimatedWeight updates max billable weight

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19535)

## Summary

As estimated weights come in on approved HHGs, the entitlements.authorized_weight (database)/ max billable weight (front end) should be updated to be 110% of the total estimated weight. This can be adjusted up or down by office users.

### How to test

1. Create a new move with a few HHGs
2. Approve as SC, then as a TOO
3. As PRIME, update a shipment to have an estimated weight
4. Login as TOO and navigate to move task order page. The Max billable weight should align with the 110% of estimated weight. Click the pencil and adjust the Max billable weight and confirm that the adjustment persists.
5. As PRIME, create a new shipment with an estimated weight.
6. Login as TOO and navigate to move task order page. The Max billable weight should align with the 110% of estimated weight.
